pipelines/destroy.yml

```diff
@@ -2,32 +2,37 @@
 resource_types:
   - name: s3-iam
     type: docker-image
+    check_every: 24h
     source:
       repository: governmentpaas/s3-resource
       tag: fda60bf4c5f85e96c16f704e128e5ead9e84d30d
 
   - name: semver-iam
     type: docker-image
+    check_every: 24h
     source:
       repository: governmentpaas/semver-resource
       tag: ecbdd201e122b44de99a40ac9f24407c1a43b9a2
 
 resources:
   - name: pipeline-trigger
     type: semver-iam
+    check_every: 24h
     source:
       bucket: ((state_bucket_name))
       region_name: ((aws_region))
       key: ((pipeline_trigger_file))
 
   - name: paas-release-ci
     type: git
+    check_every: 10h
```

Comment:
Did you mean to use 10 hours here. I think that would be easy to miss, It's probably better to stick to wither 10m or 24h everywhere.

Reply:
sorry, I meant 10m